From d3deb01456d7ed102e49fec4fdb982ebbe2c0272 Mon Sep 17 00:00:00 2001
From: Zhou Furong <furong.zhou@intel.com>
Date: Sat, 2 Feb 2019 10:44:05 +0800
Subject: [PATCH 258/292] Fix in-compatible size_t print format for i386/x84_64

print format for size_t %u and %lu can't match both
i386 and x86_64 arch. use %zu instead.

Change-Id: I942ce46abdd66a524d65342772259cb6ddc55778
Tracked-On: PKT-1719
Signed-off-by: Zhou Furong <furong.zhou@intel.com>
---
 security/keystore/api_dal.c                 | 4 ++--
 security/keystore/appauth/manifest_parser.c | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/security/keystore/api_dal.c b/security/keystore/api_dal.c
index e1d34ae1f2cd..a7f782f4e580 100644
--- a/security/keystore/api_dal.c
+++ b/security/keystore/api_dal.c
@@ -133,7 +133,7 @@ static int dal_keystore_register_client(enum keystore_seed_type seed_type,
 
 	res = handle_command_response(res, response_code, &retry, NULL, NULL);
 	if (res) {
-		ks_info(KBUILD_MODNAME ": %s Error in send_and_receive: command id = %d %d %lu\n",
+		ks_info(KBUILD_MODNAME ": %s Error in send_and_receive: command id = %d %d %zd\n",
 					 __func__, commandId, res,
 					 response_code);
 
@@ -563,7 +563,7 @@ int dal_keystore_wrap_key(const uint8_t *client_ticket,
 	kzfree(input);
 
 	if (res) {
-		ks_err(KBUILD_MODNAME ": %s Error in send_and_receive: command id = %d %d %lu\n",
+		ks_err(KBUILD_MODNAME ": %s Error in send_and_receive: command id = %d %d %zd\n",
 			__func__, commandId, res, response_code);
 
 		goto exit;
diff --git a/security/keystore/appauth/manifest_parser.c b/security/keystore/appauth/manifest_parser.c
index abe9faf5de11..d3444828dccc 100644
--- a/security/keystore/appauth/manifest_parser.c
+++ b/security/keystore/appauth/manifest_parser.c
@@ -120,7 +120,7 @@ int mf_init_file_list_ctx(const uint8_t *mf,
 			(uint8_t *) (data->app_name + data->app_name_len);
 			ks_debug("DEBUG_APPAUTH: num_files_left = %d\n",
 				 ctx->num_files_left);
-			ks_debug("DEBUG_APPAUTH: bytes_left = %lu\n",
+			ks_debug("DEBUG_APPAUTH: bytes_left = %zd\n",
 				 ctx->bytes_left);
 			return ctx->num_files_left;
 		}
-- 
2.21.0

